goss-testing/scripts/postgres_replication_lag.sh

```diff
@@ -1,19 +1,56 @@
 #!/bin/bash
 
 print_results=0
-while getopts ph stack
+while getopts pm:a:w:h stack
 do
     case "${stack}" in
           p) print_results=1;;
-          h) echo "usage: postgres_replication_lag.sh           # Only print 'PASS' upon success"
+          m) POSTGRES_MAX_LAG=$OPTARG;;
+          a) POSTGRES_MAX_ATTEMPTS=$OPTARG;;
+          w) POSTGRES_WAIT_SECONDS_BETWEEN_ATTEMPTS=$OPTARG;;
+          h) echo "usage: postgres_replication_lag.sh           # Print 'OK' and 'PASS' upon success"
              echo "       postgres_replication_lag.sh -p        # Print all results and errors if found. Use for manual check."
+             echo "       postgres_replication_lag.sh -p -m <max_allowed_lag> -a <number_attempts> -w <wait_seconds_between_attempts>"
+             echo "default parameters: -m 0  -a 10  -w 10"
              exit 3;;
-         \?) echo "usage: postgres_replication_lag.sh           # Only print 'PASS' upon success"
+         \?) echo "usage: postgres_replication_lag.sh           # Print 'OK' and 'PASS' upon success"
              echo "       postgres_replication_lag.sh -p        # Print all results and errors if found. Use for manual check."
+             echo "       postgres_replication_lag.sh -p -m <max_allowed_lag> -a <number_attempts> -w <wait_seconds_between_attempts>"
+             echo "default parameters: -m 0  -a 10  -w 10"
              exit 3;;
     esac
 done
 
+# The POSTGRES_MAX_LAG may be exported by the user to control
+# the maximum lag value permitted by this script. Setting its value to
+# a negative number or a non-integer value has the effect of skipping the
+# maximum lag check. In other words, if one wishes to skip this check,
+# one could:
+# export POSTGRES_MAX_LAG=skip
+POSTGRES_MAX_LAG=${POSTGRES_MAX_LAG:-'0'}
+
+# POSTGRES_MAX_ATTEMPTS specifies the maximum number of times the
+# postgres check will be performed on a given cluster before failing. Note that
```

Comment:
"postgres" -> "PostgreSQL" -- PostgreSQL is another company's trademark we should use it as they request.

This applies throughout this script.

Reply:
Changed to "PostgreSQL"